src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -516,13 +516,17 @@ class ZipfianInt64Generator : public Generator<int64_t> {
 // by JoinGenerator today. See ChooseStringGenerator.
 class ChooseGenerator : public Appendable {
 public:
-    // constructore defined at bottom of the file to use other symbol
+    // constructor defined at bottom of the file to use other symbol
     ChooseGenerator(const Node& node, GeneratorArgs generatorArgs);
     Appendable& choose() {
+        if (_deterministic) {
+            ++_elemNumber;
+            return *_choices[_elemNumber % _choices.size()];
+        }
         // Pick a random number between 0 and sum(weights)
         // Pick value based on that.
         auto distribution = boost::random::discrete_distribution(_weights);
-        return (*_choices[distribution(_rng)]);
+        return (*_choices[distribution(_rng)]); 
```

Comment:
This looks like extra whitespace; please delete it (I'm not sure whether there's a cpp linter for genny or not)

Reply:
Done.